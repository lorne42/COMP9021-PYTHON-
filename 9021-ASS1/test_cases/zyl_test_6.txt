Yesterday, I visited Sirs Ashe and Darius. I asked Sir Ashe 
who he was, and he answered impatiently: 
"Exactly one of Sir Darius and Sir Jhin is a Knight."
 Then I met Sir Garen who introduced me to his wife
and told me: "Sir Ashe or Sir Jhin is a Knave."
 "Sir Darius, Sir Ashe and Sir Jhin are Knights," whispered Sir Garen. 
Should I trust them? 
