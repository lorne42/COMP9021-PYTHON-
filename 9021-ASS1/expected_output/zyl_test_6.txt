The Sirs are: Ashe Darius Garen Jhin
There is a unique solution:
Sir Ashe is a Knight.
Sir Darius is a Knave.
Sir Garen is a Knave.
Sir Jhin is a Knight.